$OpenBSD: patch-src_Makefile,v 1.1.1.1 2020/02/16 19:48:24 bcallah Exp $

Don't hardcode CC or -O2.
This is actually C11 according to clang.

Index: src/Makefile
--- src/Makefile.orig
+++ src/Makefile
@@ -3,7 +3,7 @@
 # Complie C source codes and output object files to $(BUILD_DIR), then links
 # them together to get a executable $(TARGET).
 
-CC = gcc
+CC ?= gcc
 DEFS = -D PACKAGE='"$(PACKAGE)"' \
 	   -D PACKAGE_NAME='"$(PACKAGE_NAME)"' \
 	   -D PACKAGE_VERSION='"$(PACKAGE_VERSION)"' \
@@ -27,7 +27,7 @@ LIBSSLLIBS = $(shell pkg-config --libs openssl)
 LIBSECRETFLAGS =$(shell pkg-config --cflags libsecret-1)
 LIBSECRETLIBS =$(shell pkg-config --libs libsecret-1)
 
-CFLAGS += -std=gnu99 -O2 -Wall -Iinc -Wno-deprecated-declarations \
+CFLAGS += -std=gnu11 -Wall -Iinc -Wno-deprecated-declarations \
 		  $(DEFS) \
 		  $(DBGFLAGS) \
 		  $(GTK3FLAGS) \
